contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -617,4 +617,66 @@ GRANT EXECUTE ON PROCEDURE sys.babelfish_remove_domain_mapping_entry TO PUBLIC;
 
 CREATE OR REPLACE PROCEDURE sys.babelfish_truncate_domain_mapping_table()
   AS 'babelfishpg_tsql', 'babelfish_truncate_domain_mapping_table_internal' LANGUAGE C;
-GRANT EXECUTE ON PROCEDURE sys.babelfish_truncate_domain_mapping_table TO PUBLIC;
\ No newline at end of file
+GRANT EXECUTE ON PROCEDURE sys.babelfish_truncate_domain_mapping_table TO PUBLIC;
+
+CREATE TABLE sys.babelfish_extended_properties (
+  dbid smallint NOT NULL,
+  schema_name name NOT NULL,
+  major_name name NOT NULL,
+  minor_name name NOT NULL,
+  type sys.varchar(50) NOT NULL,
+  name sys.sysname NOT NULL,
+  orig_name sys.sysname NOT NULL,
+  value sys.sql_variant,
+  PRIMARY KEY (dbid, type, schema_name, major_name, minor_name, name)
+);
+GRANT SELECT on sys.babelfish_extended_properties TO PUBLIC;
```

Comment:
By granting SELECT to PUBLIC, what happens to user X trying to see attributes on an object owned by a different user? SQL Server allows you to see only attributes on objects which you have permissions on

Reply:
Yes, the base table is visible to everyone. However, most base tables in Babelfish are visible to everyone, too. I have the impression that it's possibly for major version upgrade.
Maybe it's not actually a problem?